src/main/java/com/textmessenger/service/MessageServiceImpl.java

```diff
@@ -26,51 +21,43 @@
   private final UserRepository userRepository;
   private final DialogRepository dialogRepository;
   private final NotificationService notificationService;
-  private final MessageMapper messageMapper;
-  private final UserMapper userMapper;
-  private final DialogMapper dialogMapper;
 
   public MessageServiceImpl(MessageRepository messageRepository,
                             DialogRepository dialogRepository,
                             UserRepository userRepository,
-                            NotificationService notificationService,
-                            MessageMapper messageMapper,
-                            UserMapper userMapper,
-                            DialogMapper dialogMapper) {
+                            NotificationService notificationService) {
 
     this.messageRepository = messageRepository;
     this.dialogRepository = dialogRepository;
     this.userRepository = userRepository;
     this.notificationService = notificationService;
-    this.messageMapper = messageMapper;
-    this.userMapper = userMapper;
-    this.dialogMapper = dialogMapper;
   }
 
   @Override
-  public void createMessage(MessageRxDto message) {
-    Message save = messageRepository.save(messageMapper.messRxDtoToMess(message));
+  public void createMessage(Message message) {
+    Message save = messageRepository.save(message);
     User user = save.getUser();
     save.getDialog().getUsers().forEach(u -> {
       if (u.getId() != user.getId()) {
-        notificationService.createNotification(NotificationType.MESSAGE.toString(), userMapper.userToRxDto(u), save.getId());
+        notificationService.createNotification(NotificationType.MESSAGE.toString(), u, save.getId());
       }
     });
   }
 
   @Override
-  public void updateMessage(MessageRxDto message) {
-    messageRepository.save(messageMapper.messRxDtoToMess(message));
+  public void updateMessage(Message message) {
+    messageRepository.save(message);
   }
 
   @Override
-  public void deleteMessage(MessageRxDto message) {
-    messageRepository.deleteById(messageMapper.messRxDtoToMess(message));
+  public void deleteMessage(Message message) {
+    messageRepository.deleteById(message);
   }
 
   @Override
-  public List<MessageTxDto> getMessagesFromDialog(DialogRxDto dialog) {
-    return messageMapper.messsToMessTxDtos(messageRepository.findByDialog(dialogMapper.dialRxDtoToDial(dialog)));
+  public List<MessageToFront> getMessagesFromDialog(Dialog dialog) {
+    //return messageRepository.findByDialog(dialog);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=158&issues=AWWU5_bVAJjEPbGfqJZR&open=AWWU5_bVAJjEPbGfqJZR)

[ISSUE_KEY]: AWWU5_bVAJjEPbGfqJZR